requirements.txt

```diff
@@ -0,0 +1,3 @@
+cfn-lint>=0.37.0
+black>=20.8b1
+flake8>=3.8.4
```

Comment:
Ahhhh python.  I have the same question/comment here - should these be locked to an exact version?

Reply:
These are using a `>=-`, which I think is like `^` in npm. But I don't really know anything about requirements files, this is just how I've always seen it done.